### PR TITLE
feature/models

### PR DESCRIPTION
**Pull request by davidmhatton**

### CODE REVIEW

**Review by nathj07 — COMMENTED** [review]
I don't think you want to check in *.class files. They are built from the .java files.

File: src/main/java/enums/Color.java [hunk 1]
```diff
@@ -0,0 +1,32 @@
+package enums;
+
+import com.fasterxml.jackson.annotation.JsonValue;
+
+/**
+ * Description.
```
Comment (nathj07) [inline src/main/java/enums/Color.java]: I would think a top level comment could provide all the necessary detail and then the comments around the enum values are somewhat redundant

File: src/test/java/models/places/SquareTest.java [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+import org.junit.runner.RunWith;
+import org.junit.runners.BlockJUnit4ClassRunner;
+
+import enums.Color;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+/**
+ * Test package for squares.
+ * NERD!
```
Comment (nathj07) [inline src/test/java/models/places/SquareTest.java]: what does this comment add?

**Review by nathj07 — COMMENTED** [review]
overall looks alright to me. It's been a while since I did any Java so I may not be up on the latest practices and idioms

File: src/main/java/enums/Color.java (hunk 1 above)
Comment (davidmhatton) [reply src/main/java/enums/Color.java]: IDE default - forgot it was there.

File: src/test/java/models/places/SquareTest.java (hunk 2 above)
Comment (davidmhatton) [reply src/test/java/models/places/SquareTest.java]: Very little aside from humour.

**Pull request merged**